docs/concepts/gardenlet.md

```diff
@@ -1,119 +1,137 @@
 
-Basically, it is possible to define settings for the Kubernetes clients interacting with the various clusters, settings for the control loops inside the Gardenlet, settings for leader election and log levels, feature gates, and seed selection/configuration.
+* settings for the Kubernetes clients interacting with the various clusters
+* settings for the control loops inside the gardenlet
+* settings for leader election and log levels, feature gates, and seed selection or seed configuration.
 
-Most of the configuration options are similar to what the gardener-controller-manager offered with previous Gardener versions.
+More information: [Example Gardenlet Component Configuration](../../example/20-componentconfig-gardenlet.yaml).
 
 ## Heartbeats
 
-Similar to how Kubernetes is meanwhile using `Lease` objects for node heart beats (see [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/0009-node-heartbeat.md)), the Gardenlet is using `Lease`s objects for seed heart beats.
-Every two seconds it is checking its connectivity to the seed and then it renews its lease.
-The status will be reported in the `GardenletReady` condition in its `Seed` object(s).
-Similarly to the `node-lifecycle-controller` inside the kube-controller-manager, the gardener-controller-manager features a `seed-lifecycle-controller` that will set the `GardenletReady` condition to `Unknown` in case the Gardenlet stops sending its heartbeat signals.
-Carrying on, this will make the gardener-scheduler not considering this seed for newly created shoots anymore.
+Similar to how Kubernetes is meanwhile using `Lease` objects for node heart beats (see [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/0009-node-heartbeat.md)), the gardenlet is using `Lease` objects for heart beats of the seed cluster.
+Every two seconds, gardenlet is checking its connectivity to the seed cluster and then it renews its lease.
+The status is reported in the `GardenletReady` condition in its `Seed` object list.
+Similarly to the `node-lifecycle-controller` inside the `kube-controller-manager`, the `gardener-controller-manager` features a `seed-lifecycle-controller` that is set the `GardenletReady` condition to `Unknown` in case the gardenlet stops sending its heartbeat signals.
+As a consequence, the `gardener-scheduler` doesn’t create new shoot clusters in such a seed cluster anymore.
 
 ### `/healthz` Endpoint
 
-[gardener/gardener#2309](https://github.com/gardener/gardener/pull/2309) has enhanced the Gardenlet with a HTTPS server that serves a `/healthz` endpoint.
-It is used as a liveness probe in the `Deployment` of the Gardenlet.
-If the Gardenlet fails trying to renew its lease then the endpoint will return `500 Internal Server Error`, otherwise it will return `200 OK`.
+The gardenlet includes an HTTPS server that serves a `/healthz` endpoint. It’s used as a liveness probe in the `Deployment` of the gardenlet.
```

Comment:
```suggestion
The gardenlet includes a HTTPS server that serves a `/healthz` endpoint. It’s used as a liveness probe in the `Deployment` of the gardenlet.
```

Reply:
@rfranzke 
It's not just about the letter that determines when to use "a" or "an", it's about the *vowel sound* in the beginning of the following word.

So although "HTTP" starts with a consonant, it's simply hard to say "a HTTPS server", because when you say it, you end up saying (phonetically): "a eitsch-te-te-pe ...", so having to vowels in a row. 

That's why "an HTTPS server" is correct here.